deepspeed/checkpoint/deepspeed_checkpoint.py

```diff
@@ -116,7 +116,7 @@ def show_transformer_file_map(self):
         self._dump_mapping(self.transformer_file_map, 'rank_to_transformer_files')
 
     def _build_global_state(self):
-        sd = torch.load(self.mp_rank_files[0], map_location=torch.device('cpu'))
```

Comment:
Hi @terry-for-github - could you run the pre-commit formatter, that should fix the formatting errors on these files.

Reply:
ok, I'll try that.